test/ThriftTest.thrift

```diff
@@ -409,3 +409,8 @@ struct StructB {
   1: optional StructA aa;
   2: required StructA ab;
 }
+
+service OneWayTest {
```

Comment:
Could we move OneWayTest to its own thrift file, which could be located inside lib/cpp/test because it is only used for that unit test?  I can help in making sure the Makefile.am and the CMakeLists.txt are updated appropriately.